docs/DeveloperGuide.md

```diff
@@ -7,13 +7,62 @@ title: Developer Guide
+It aims to contain all the information required to quickly get a calendar UI into your application. 
+
+In the [Table of Contents](#), each item listed is a link which you can click on to go directly to that section in the guide.
+
+**Conventions Used**
+
+Syntax | Meaning
+--------|------------------
+`text` | A command, or a code block
+<kbd>text</kbd> | A keyboard input, or a button to be clicked on.
+[text](#about-this-guide) | Links to other parts of the document, or links to be opened in the browser.
+_text_ | A _technical_ word with definitions provided in the [Glossary](#appendix-b-glossary), or a caption for images.
+<div markdown="block" class="alert alert-info"> :information_source: </div> | Indication that the following text is a note, which is useful in helping you understand how TAB works.
+<div markdown="block" class="alert alert-primary"> :bulb: </div> | Indication that the following text is a tip, which allows you to interact with TAB more effectively and quickly.
+<div markdown="block" class="alert alert-warning"> :exclamation: </div> | Indication that the following text is important. Missing it out may impair your user experience and may potentially cause TAB to misbehave!
+
+--------------------------------------------------------------------------------------------------------------------
+
 ## **Acknowledgements**
 
 List of sources of all reused/adapted ideas, code, documentation, and third-party libraries:
 
 * This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).
 * Libraries used: [CalendarFX](https://dlsc.com/products/calendarfx/), [Jackson](https://github.com/FasterXML/jackson), [JavaFX](https://openjfx.io/), [JUnit5](https://github.com/junit-team/junit5)
-* Table cells with copy and paste and wrap-text functions reference: [Roland09](https://gist.github.com/Roland09/6fb31781a64d9cb62179#file-tableutils-java), [James_D](https://stackoverflow.com/questions/22732013/javafx-tablecolumn-text-wrapping).
+* The `SchedulePanel#createTimeThread()` method was reused with minimal changes from the CalendarFX [developer manual](http://dlsc.com/wp-content/html/calendarfx/manual.html#_quick_start).
+
+```java
+private void createTimeThread() {
+    Thread updateTimeThread = new Thread("Calendar: Update Time Thread") {
+        @Override
+        public void run() {
+            while (true) {
+                Platform.runLater(() -> {
+                    calendarView.setToday(LocalDate.now());
+                    calendarView.setTime(LocalTime.now());
+                });
+                try {
+                    // update every 60 seconds
+                    sleep(60000);
+                } catch (InterruptedException e) {
+                    e.printStackTrace();
+                }
+            }
+        }
+    };
+    updateTimeThread.setPriority(Thread.MIN_PRIORITY);
+    updateTimeThread.setDaemon(true);
+    updateTimeThread.start();
+}
+```
```

Comment:
would a link be better instead? To have a code snippet in acknowledgments is pretty weird